media-gfx/blender/blender-9999.ebuild

```diff
@@ -59,6 +59,7 @@ RDEPEND="${PYTHON_DEPS}
 	media-libs/libjpeg-turbo:=
 	media-libs/libpng:=
 	media-libs/libsamplerate
+	>=media-libs/openimageio-2.4.6.0:=
```

Comment:
I'm not asking you to bother redoing the commits but I dare say the diffs would be much smaller if you made this first and then the commit adding OpenPGL.

Reply:
I can probably reorder the commits with a simple rebase if you would prefer that.